doc/dev/crimson/crimson.rst

```diff
@@ -182,6 +182,23 @@ using ``vstart.sh``,
 ``--memstore``
     use the alienized MemStore as the object store backend.
 
+``--seastore``
+    use the SeaStore as object store backend.
+
+``--seastore-devs``
+    specify the block device used by SeaStore.
+
+``--seastore-secondary-devs``
+    Optional.  SeaStore supports multiple devices.  Enable this feature by
+    passing the block device to this option.
+
+``--seastore-secondary-devs-type``
+    Optional.  Specify device type of secondary devices.  When the secondary
+    device is slower than main device passed to ``--seastore-devs``, the cold
+    data in faster device will be evicted to the slower devices over time.
+    Valid args include: HDD, SSD(default), ZNS and RANDOM_BLOCK_SSD.
```

Comment:
Uppercase CLI arguments are unusual but I don't think I can ask you to lowercase them in the code.

Suggest backticks around the literals, and a brief explantion of how `SSD` and `RANDOM_BLOCK_SSD` differ; on the surface the two seem to be identical.

Valid types include \`\`HDD\`\`, \`\`SSD\`\`(default), \`\`ZNS\`\`, and \`\`RANDOM
_BLOCK_SSD\`\``